### DISCUSSION TITLE
Is protonAOSP allowed?

### DISCUSSION THREAD

**Discussion by ibratabian17** [ROM-builders/temporary#8520] [Q&A] [answered]
.

**Comment by Apon77** [comment, marked as answer]
> .

https://github.com/ROM-builders/temporary/blob/main/test.sh#L92

**Reply by ibratabian17** [reply]
> > .
> 
> https://github.com/ROM-builders/temporary/blob/main/test.sh#L92

Ok, but, when ArrowOS-12.1 added?
